pkgs/development/libraries/openvdb/11.nix

```diff
@@ -10,9 +12,26 @@ openvdb.overrideAttrs (old: rec {
   src = fetchFromGitHub {
     owner = "AcademySoftwareFoundation";
     repo = "openvdb";
-    rev = "v${version}";
-    sha256 = "sha256-wDDjX0nKZ4/DIbEX33PoxR43dJDj2NF3fm+Egug62GQ=";
+    tag = "v${version}";
+    hash = "sha256-wDDjX0nKZ4/DIbEX33PoxR43dJDj2NF3fm+Egug62GQ=";
   };
+
+  patches = lib.optionals stdenv.cc.isClang [
+    # Fix Clang/LLVM build issues, originally resolved in version 12.0.1:
+    # https://github.com/AcademySoftwareFoundation/openvdb/pull/1977
+    (fetchpatch {
+      name = "fix-clang-template-errors.patch";
+      url = "https://github.com/AcademySoftwareFoundation/openvdb/commit/e3ee8508ac3b3ff932859652f88f4e524c656be2.patch";
```

Comment:
Commit should be https://github.com/AcademySoftwareFoundation/openvdb/commit/930c3acb8e0c7c2f1373f3a70dc197f5d04dfe74, we should be using the merged commit and not the PR's commit.

Reply:
Thanks! got it